spec/active_job/queue_adapters/sidekiq_publisher_adapter_spec.rb

```diff
@@ -41,7 +41,7 @@ def perform(*args); end
       end
     end
 
-    context "when not in a transaction", skip_db_clean: true do
+    context "when not in a transaction", run_outside_transaction: true do
```

Comment:
Since we are going to be creating database records outside of a transaction I refactored this to be more explicit about the intent and to truncate.